src/main/groovy/life/qbic/utils/DataSetValidationException.groovy

```diff
@@ -0,0 +1,41 @@
+package life.qbic.utils
+
+import org.everit.json.schema.ValidationException
+
+/**
+ * Exception that shall be thrown when a dataset validation failed.
+ *
+ * This exception class wraps the underlying <code>org.everit.json.schema
+ * .ValidationException</code> class, to encapsulate this third party dependency.
+ *
+ * @since 1.7.0
+ */
+class DataSetValidationException extends RuntimeException {
+
+    private final ValidationException validationException
+
+    DataSetValidationException(ValidationException validationException) {
+        this.validationException = validationException
+    }
+
+    /**
+     * Returns all messages of found dataset validations.
+     * @return A list of exception messages
```

Comment:
```suggestion
     * Returns all messages of detected schema violations.
     * @return A list of validation violation messages
     * @since 1.7.0
```